### PR TITLE
Add shell script that build a release of the plugin

### PR DESCRIPTION
**Pull request by dlumbrer**
The shell script is the same that the one that was in the `make-release-6.1.0-10` changing a few things:

- The release now is in the 6.1.0-12 version
- The number of the version (The X of 6.1.0-12-X) is now provided by an argument

### CODE REVIEW

File: release_biter.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# Check if param exist
+if [ $# -eq 0 ]
+  then
+    echo "error: No arguments supplied" >&2
+    echo "Usage: ./release_biter.sh <n_version>" >&2; exit 1
+fi
+
+# Check if the param is a number
+re='^[0-9]+$'
```
Comment (sanacl) [inline release_biter.sh]: Nicely done!

File: release_biter.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+if ! [[ $1 =~ $re ]] ; then
+   echo "error: <n_version> is not a number" >&2; exit 1
+fi
+
+# Create structure
+mkdir -p releases
+mkdir -p releases/searchguard
```
Comment (sanacl) [inline release_biter.sh]: We don't need line 17 due to the "-p" flag

File: release_biter.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# Copy dependencies
+cp -rf lib/ releases/searchguard/.
+cp -rf node_modules/ releases/searchguard/.
+cp -rf public/ releases/searchguard/.
+cp index.js releases/searchguard/.
+cp package.json releases/searchguard/.
+
+# Build zip
+cd releases
+[ -e searchguard-biter-6.1.0-12-$1.zip ] && rm searchguard-biter-6.1.0-12-$1.zip
```
Comment (sanacl) [inline release_biter.sh]: Please avoid single line conditionals. I would :heart: to get the code as simple as possible

Comment (dlumbrer) [reply release_biter.sh]: :+1:

**Pull request merged**